daq/faucet_states_collector.py

```diff
@@ -239,9 +243,17 @@ def process_config_change(self, timestamp, dp_name, restart_type, dp_id, dps_con
             dp_state[KEY_DP_ID] = dp_id
             dp_state[KEY_CONFIG_CHANGE_TYPE] = restart_type
             dp_state[KEY_CONFIG_CHANGE_TS] = datetime.fromtimestamp(timestamp).isoformat()
-            self.system_states[DPS_CONFIG] = dps_config
             dp_state[KEY_CONFIG_CHANGE_COUNT] = dp_state.setdefault(KEY_CONFIG_CHANGE_COUNT, 0) + 1
 
+    @dump_states
+    def process_config_change(self, timestamp, dps_config):
```

Comment:
process dps_config_change?    How is this distinguished from some other kind of config change?

Reply:
if we don't want to use "dps" since it's too close to "dp"-- maybe the full dataplane?  As in process_dataplane_config_change?  Not sure if that's too wordy or not.